packages/l/llvm/xmake.lua

```diff
@@ -23,30 +23,56 @@ package("llvm")
     add_configs("libcxxabi",         {description = "Enable libcxxabi runtime.", default = true, type = "boolean"})
     add_configs("openmp",            {description = "Enable openmp runtime.", default = false, type = "boolean"})
 
+    -- before adding a new version, make sure that the prebuild is containing llvm-config
     if on_source then
         on_source(function (package)
+            import("core.base.semver")
             local precompiled = false
-            if package:is_plat("windows") then
-                if package:is_arch("x86") then
-                    package:set("urls", "https://github.com/xmake-mirror/llvm-windows/releases/download/$(version)/clang+llvm-$(version)-win32.zip")
-                    package:add("versions", "11.0.0", "268043ae0b656cf6272ccb9b8e3f21f51170b74ed8997ddc0b99587983b821ca")
-                    package:add("versions", "14.0.0", "63afc3c472cb279978c5a7efc25b8783a700aeb416df67886b7057eba52a8742")
-                    package:add("versions", "15.0.7", "8dbabb2194404220f8641b4b18b24b36eca0ae751380c23fc7743097e205b95f")
-                    package:add("versions", "16.0.6", "5e1f560f75e7a4c7a6509cf7d9a28b4543e7afcb4bcf4f747e9f208f0efa6818")
-                    package:add("versions", "17.0.6", "ce78b510603cb3b347788d2f52978e971cf5f55559151ca13a73fd400ad80c41")
-                    package:add("versions", "18.1.1", "9f59dd99d45f64a5c00b00d27da8fe8b5f162905026f5c9ef0ade6e73ae18df3")
-                    precompiled = true
+            local latest = "18.1.8"
+
+            local requiredversion = (package:requireinfo() and package:requireinfo().version) or "latest" == "latest"
```

Comment:
"latest" == "latest"?

Reply:
you can refer https://github.com/xmake-io/xmake/blob/5e865e25fc58773ab0194084b6f6506f463e68e4/xmake/modules/private/action/require/impl/package.lua#L442